pkgs/by-name/ne/neovim-node-client/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchNpmDeps,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage rec {
+  pname = "node-client";
+  version = "5.1.1-dev.0";
```

Comment:
I think you can push a new tag named like that to the source repo, right?

Otherwise we would need to convert it to 5.1.0-unstable-XXXX-XX-XX and fill in the date of the commit.